client/my-sites/plans/jetpack-plans/product-lightbox/payment-plan.tsx

```diff
@@ -19,23 +19,24 @@ type PaymentPlanProps = {
 	product: SelectorProduct;
 	showPlansOneBelowTheOther?: boolean;
 	isActive?: boolean;
+	quantity?: number | null;
 };
 const PaymentPlan: React.FC< PaymentPlanProps > = ( {
 	isMultiSiteIncompatible,
 	siteId,
 	product,
 	showPlansOneBelowTheOther = false,
 	isActive = true,
+	quantity = null,
 } ) => {
 	const translate = useTranslate();
 	const { containerRef, isCompact } = useItemPriceCompact();
 
-	const { originalPrice, discountedPrice, discountedPriceDuration, isFetching } = useItemPrice(
-		siteId,
-		product,
-		product?.monthlyProductSlug || ''
-	);
+	const { originalPrice, discountedPrice, discountedPriceDuration, isFetching, priceTierList } =
+		useItemPrice( siteId, product, product?.monthlyProductSlug || '' );
 
+	const currentTier = quantity && priceTierList.find( ( tier ) => tier.maximum_units === quantity );
+	const currentTierPrice = currentTier && currentTier.minimum_price / 100 / 12;
```

Comment:
I assume the division by 12 is just for annual plans here.
I think that's probably OK for now - as long as we don't shift to showing other plan terms with this code 😅

Reply:
It's probably find since we show annual prices everywhere. I don't see that changing anytime soon 😄